klog_test.go

```diff
@@ -1754,3 +1754,53 @@ func TestKlogFlagPrefix(t *testing.T) {
 		}
 	})
 }
+
+func TestKObjs(t *testing.T) {
+	tests := []struct {
+		name string
+		obj  interface{}
+		want []ObjectRef
+	}{
+		{
+			name: "test for KObjs function with KMetadata slice",
+			obj: []kMetadataMock{
```

Comment:
Can you add a test case with pointer to kMedatadaMock? Would be great of one of the elements was nil.